src/content/docs/es/plugin/notification.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: Notificaciones
+description: Envía notificaciones nativas al usuario.
```

Comment:
Have you consider using a more inclusive language? I don't know if there are content or translation guidelines related in the project.

Here's a commonly cited [resource](https://ajuntament.barcelona.cat/guia-de-llenguatge-no-sexista/es/). One alternative to "usuario" could be "persona usuaria" o "persona destinataria".

Needless to say, there are several ocurrences of "usuario" as a masculine noun.

Reply:
That's a very valid concern, and I actually considered it, but the translation guidelines mandate that there should be consistency between pages. See https://github.com/tauri-apps/tauri-docs/blob/v2/.github/TRANSLATING.md

What we could do is to maybe use "usuario" for now and consider a changing it in all docs in a different PR. What do you think?

Thanks for your review! :pray: